File: monitor.php
<?php

$logFilePath = 'unique_client_ips.log';
$clientIP = $_SERVER['REMOTE_ADDR'];
$ipFound = false;

// Open the log file for reading
if ($file = fopen($logFilePath, "r")) {
    while (!feof($file) && !$ipFound) {
        $loggedIP = trim(fgets($file));
        if ($loggedIP == $clientIP) {
            $ipFound = true;
        }
    }
    fclose($file);
}

echo $ipFound ? 'true' : 'false';

?>

File: monitor.py
logged_ips = set()

def request(flow):
    client_ip = flow.client_conn.address[0]
    if client_ip not in logged_ips:
        logged_ips.add(client_ip)
        with open("unique_client_ips.log", "a") as log_file:
            log_file.write(f"{client_ip}\n")
